src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1

```diff
@@ -78,77 +78,179 @@ function New-AzDataMigrationLoginsMigration
             #Defining Default Output Path
             $DefaultOutputFolder = Get-DefaultLoginMigrationsOutputFolder
 
-            #Defining Base and Exe paths
-            $BaseFolder = Join-Path -Path $DefaultOutputFolder -ChildPath Downloads;
-            $ExePath = Join-Path -Path $BaseFolder -ChildPath Logins.Console.csproj\Logins.Console.exe;
+            #Defining Downloads folder
+            $DownloadsFolder = Join-Path -Path $DefaultOutputFolder -ChildPath Downloads;
 
-            #Checking if BaseFolder Path is valid or not
-            if(-Not (Test-Path $BaseFolder))
+            #Checking if DownloadsFolder Path is valid or not
+            if(-Not (Test-Path $DownloadsFolder))
             {
-                $null = New-Item -Path $BaseFolder -ItemType "directory"
+                $null = New-Item -Path $DownloadsFolder -ItemType "directory"
+            }
+            else
```

Comment:
Can there be any other folders/files in the old `$DownloadsFolder`? If yes, do we want to clean the whole folder and recreate it instead of individually deleting the files/folders?

Reply:
There could be a console app that was earlier downloaded, and we don't want to delete and download it in each iteration. 
As part of the change, I have renamed the console app from Login.console to Microsoft.SqlServer.Migration.Logins.ConsoleApp to keep it consistent with all other NuGets. The check here is to delete the console app named "Login.console" and all its extracted files.